drivers/flash/flash_stm32.c

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2017 Linaro Limited
+ * Copyright (c) 2017 BayLibre, SAS.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <string.h>
+#include <flash.h>
+#include <init.h>
+#include <soc.h>
+
+#include <flash_stm32.h>
+
+#if defined(CONFIG_SOC_SERIES_STM32L4X)
+#define FLASH_KEY1	((uint32_t)0x45670123U) /* Flash key1 */
+#define FLASH_KEY2	((uint32_t)0xCDEF89ABU)	/* Flash key2 */
```

Comment:
Yes, but not for L4x:

> $ grep FLASH_KEY ext/hal/st/stm32cube/stm32f*/soc/ -R | wc -l
> 336

They are defined in the stm32f*****.h files.

> $ grep FLASH_KEY ext/hal/st/stm32cube/stm32l*/soc/ -R | wc -l
> 0

They are defined in stm32cube/stm32l4xx/drivers/include/stm32l4xx_hal_flash.h

How should that be handled ?